frontends/web/src/routes/exchange/btcdirect.tsx

```diff
@@ -94,15 +94,45 @@ export const BTCDirect = ({ accounts, code }: TProps) => {
     }, 200);
   };
 
+  const locale = i18n.resolvedLanguage ? localeMapping[i18n.resolvedLanguage] : 'en-GB';
+
+  const onMessage = useCallback((event: MessageEvent) => {
+    // if (!isDevserver && event.origin !== 'https://shiftcrypto.io') { return; }
```

Comment:
Why is this commented?

Reply:
I'll update once we have the backend part, the correct subdomain is `https://bitboxapp.shiftcrypto.io`.

If you don't mind I'll continue in https://github.com/BitBoxSwiss/bitbox-wallet-app/pull/3152

Reply:
updated in #3152